nifi-nar-bundles/nifi-azure-bundle/pom.xml

```diff
@@ -29,6 +29,8 @@
         <azure-storage.version>8.4.0</azure-storage.version>
         <azure.core.version>1.5.0</azure.core.version>
         <jackson.version>2.10.3</jackson.version>
+        <azure-cosmos.version>4.2.0</azure-cosmos.version>
+        <azure-core.version>1.6.0</azure-core.version>
```

Comment:
If we stay with azure core 1.5.0, this will not work with Cosmos 4.2.0, which has the better performance than Cosmos 4.1.0.

I tested out integration tests with azure core version 1.6.0, and it shows the same test result as 1.5.0. No test failures with azure-datalake processors.

Reply:
com.azure:azure-identity (https://mvnrepository.com/artifact/com.azure/azure-identity/1.0.6) has also dependency on com.azure:azure-core, which is 1.5.0 and is upgradable to 1.9.0. Thus, it is also not breaking with 1.6.0 azure-core version.